modules/custom/az_news/az_news_feeds/az_news_feeds.info.yml

```
@@ -0,0 +1,14 @@
+name: Quickstart News Feeds
+type: module
+description: 'Import news stories from UArizona News.'
+core_version_requirement: ^8.8 || ^9
+package: 'The University of Arizona'
+
+dependencies:
+  - az_news
```

I think should add `az_core` since we use its permission.
```suggestion
  - az_core
  - az_news
```